api/v1alpha1/ndb_api_helpers.go

```diff
@@ -269,7 +269,12 @@ func GetActionArgumentsByDatabaseType(databaseType string) (DatabaseActionArgs,
 }
 
 func (m *MysqlActionArgs) GetActionArguments(dbSpec DatabaseSpec) []ActionArgument {
-	return []ActionArgument{}
+	return []ActionArgument{
```

Comment:
Can you add a new UT or update an existing UT which tests the argument generation for MySQL?